autovalet_navigation/config/sim/global_costmap_params.yaml

```diff
@@ -1,13 +1,24 @@
+# Added lane as a plugin for obstacle layer (?)
 global_costmap:
   plugins:
       - {name: static_layer,   type: "costmap_2d::StaticLayer"}
       - {name: inflation,      type: "costmap_2d::InflationLayer"}
+      - {name: lane,           type: "costmap_2d::ObstacleLayer"}
+
+  lane:
```

Comment:
why does this not live in the costmap_common_params.yaml file next to the other pcl sources?

Reply:
It didn't work if I put it in costmap common. Had to be here for it to work